tests/dummy/app/routes/index.js

```diff
@@ -14,12 +16,22 @@ const basicObject = {
         "title": "Ranking",
         "enum": ['excellent', 'not too shabby', 'alpaca built my hotrod']
       }
-    },
+    }
   }
 };
 
 const basicString = JSON.stringify(basicObject);
 
+const basicFilterRule = _.clone(basicObject, true);
+
+basicFilterRule["options"] = {
+  "fields": {
+    "name": {
+      "filterrules": [ "admin-only" ]
```

Comment:
I think to follow conventions we should rename to filter-rules instead. its both a directory and a json key, both of which we dasherize.